### PR TITLE
Other texts in bavet119

### PR DESCRIPTION
**Pull request by MassimoVilla**
Unmerged branch on records about astrological works contained in BAVet119

### CODE REVIEW

File: 1001-2000/LIT1171Awdana.xml [hunk 1]
```diff
@@ -71,6 +76,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             </listBibl>
          </div>
          <div type="edition">
+            <div type="textpart" xml:id="list">
+               <label>List of names and letters with which portions of the circles are annotated</label>
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1171Awdana.xml:80]: Better: "List of names and letters used to label parts of the circles"

File: new/LIT14518Diagram.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7122Diagram" xml:lang="en" type="work">
```
Comment (CarstenHoffmannMarburg) [inline new/LIT14518Diagram.xml:4]: I understand, that LIT14518Diagram is identical to LIT7122Diagram and will be deleted in another pull request, correct?
Otherwise, xml:id must be changed, because it must correspond to the title of the file.

File: new/LIT14518Diagram.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Calendric diagram</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT14518Diagram.xml:8]: I think, it is: "Calendrical diagram"

File: new/LIT14518Diagram.xml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                        <foreign xml:lang="gez">ጰ<supplied reason="omitted">ጕሜን፡</supplied></foreign>, i.e. the epagomenal days;
+                        <foreign xml:lang="gez">ዓውደ<supplied reason="omitted">፡</supplied> ዓመት<supplied reason="omitted">፡</supplied></foreign>, i.e. the period of the year;
+                        <foreign xml:lang="gez">ዕለ<supplied reason="omitted">ተ</supplied> ዮሐን<supplied reason="omitted">ስ፡</supplied></foreign>, i.e. the day of John;
+                        <foreign xml:lang="gez">ጥንት<supplied reason="omitted">ዮን፡</supplied></foreign>, i.e. the <foreign xml:lang="gez">ṭǝntyon</foreign>;
+                        <foreign xml:lang="gez">አበቅቴ<supplied reason="omitted">፡</supplied></foreign>, i.e. the epact;
+                        <foreign xml:lang="gez">መጥቅ<supplied reason="omitted">ዕ፡</supplied></foreign>, i.e. the <foreign xml:lang="gez">maṭqǝʾ</foreign>;
+                        <foreign xml:lang="gez"><sic>ሠርቅ<supplied reason="omitted">፡</supplied> ሌለ<supplied reason="omitted">ት፡</supplied></sic></foreign>, i.e. the beginning of the night (which is "1" in all boxes);
```
Comment (CarstenHoffmannMarburg) [inline new/LIT14518Diagram.xml:70]: ሌ`<choice resp="MV"><sic>ለ</sic><corr>ሊ</corr></choice>ት፡` or `ሌ<sic>ለ</sic>ት፡`

File: new/LIT14519Winds.xml [hunk 5]
```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT14523Winds" xml:lang="en" type="work">
```
Comment (CarstenHoffmannMarburg) [inline new/LIT14519Winds.xml:4]: I understand, that LIT14519Winds is identical to LIT7123Winds and will be deleted in another pull request, correct? Otherwise the xml:id must be changed to be identical to the file name.

Comment (MassimoVilla) [reply new/LIT14519Winds.xml:4]: They can be deleted in this PR as well, so that only LIT7122Diagram 
 and LIT7123Winds are extant. Now they should have been correctly deleted, isn't it? Thank you :)

Comment (eu-genia) [reply new/LIT14519Winds.xml]: yes they are deleted on the backend already, and merging these files would recreate the numbering error, thank you for removing them and updating the xml:id

**Review by eu-genia — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Comment by MassimoVilla** [comment]
Can I merge this?

**Review by CarstenHoffmannMarburg — APPROVED** [review]
